### DISCUSSION TITLE
How to use KaTeX within HTML tags?

### DISCUSSION THREAD

**Discussion by ghost** [KaTeX/KaTeX#3920] [Q&A] [answered]
_Apologies if I'm missing something obvious, but I've looked hard and can't find a solution._

I'm using [KaTeX within Docusaurus](https://docusaurus.io/docs/markdown-features/math-equations). I'd like to put  a derivation in a `<details>` HTML tag so it can be expanded by someone who's interested but is collapsed by default and doesn't crowd the page. 

But I don't know how to make expressions render correctly within HTML tags. When I try using the same syntax with `$` or `$$` it doesn't work (unsurprisingly). Is there a special tag I can use inside of a React app like a Docusaurus project? Is there some other way I can achieve this?

Thanks!

**Comment by edemaine** [comment, marked as answer]
Did you try adding blank lines before/after the HTML tags, like so? That's how Markdown normally works (with Markdown syntax). 

```markdown
<details>

$x+y$ is math

</details>
```

**Comment by ghost** [comment]
Ah, that worked! Such a simple thing — appreciate you pointing this out.
